modules/olm-overriding-operator-pod-affinity.adoc

```diff
@@ -0,0 +1,259 @@
+spec:
+  name: my-package
+  source: my-operators
+  sourceNamespace: operator-registries
+  config:
+    affinity:
+      podAntiAffinity: <1>
+        requiredDuringSchedulingIgnoredDuringExecution:
+        - labelSelector:
+            matchExpressions:
+            - key: cpu
+              operator: In
+              values:
+              - high
+          topologyKey: kubernetes.io/hostname
+ ...
+----
+<1> A pod anti-affinity that prevents the Operator's pod from being scheduled on a node that has pods with the `cpu=high` label.
+endif::node[]
+
+.Procedure
+
+To control the placement of an Operator pod:
+
+. Install the Operator as usual.
+
+. If needed, ensure that your nodes are labeled to properly respond to the affinity.
+
+. Edit the Operator `Subscription` object to add an affinity:
++
+ifndef::pod[]
+[source,terminal]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  creationTimestamp: '2022-12-21T14:54:47Z'
+  generation: 6
+  labels:
+    operators.coreos.com/openshift-custom-metrics-autoscaler-operator.openshift-keda: ''
+  name: openshift-custom-metrics-autoscaler-operator
+  namespace: openshift-keda
+  resourceVersion: '77887'
+  uid: 9b38d0eb-9c57-4cb6-9905-1c65c21f08ef
+spec:
+  name: my-package
+  source: my-operators
+  sourceNamespace: operator-registries
+  config:
+    affinity: <1>
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+            - matchExpressions:
+                - values:
+                    - ip-10-0-185-229.ec2.internal
+                  key: kubernetes.io/hostname
+                  operator: In
+ ...
+----
+<1> Add a `nodeAffinity` or `podAntiAffinity`. See the Additional resources section that follows for information about creating the affinity.
+endif::pod[]
+ifdef::pod[]
+[source,terminal]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  creationTimestamp: '2022-12-21T14:54:47Z'
+  generation: 6
+  labels:
+    operators.coreos.com/openshift-custom-metrics-autoscaler-operator.openshift-keda: ''
+  name: openshift-custom-metrics-autoscaler-operator
+  namespace: openshift-keda
+  resourceVersion: '77887'
+  uid: 9b38d0eb-9c57-4cb6-9905-1c65c21f08ef
+spec:
+  name: my-package
+  source: my-operators
+  sourceNamespace: operator-registries
+  config:
+    affinity:
+      podAntiAffinity: <1>
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+            - matchExpressions:
+                - values:
+                    - ip-10-0-185-229.ec2.internal
+                  key: kubernetes.io/hostname
+                  operator: In
+ ...
+----
+<1> Add a `podAntiAffinity`. See the Additional resources section that follows for information about creating the affinity.
+endif::pod[]
+
+.Verification
+
+* Ensure that pod is deployed on the specific node:
++
+[source,terminal]
+----
+$ oc get pods -o wide
+----
++
+.Sample output
```

Comment:
```suggestion
.Example output
```
I think we default to _Example output_ based on the [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output).